Sources/OpenTelemetryInstrumentationSupport/SpanAttribute+RPCSemantics.swift

```diff
@@ -0,0 +1,39 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift Tracing open source project
+//
+// Copyright (c) 2020 Moritz Lang and the Swift Tracing project authors
+// Licensed under Apache License v2.0
+//
+// See LICENSE.txt for license information
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+//===----------------------------------------------------------------------===//
+
+#if swift(>=5.2)
+import TracingInstrumentation
+
+/// Semantic conventions for HTTP spans as defined in the OpenTelemetry spec.
```

Comment:
```suggestion
/// Semantic conventions for RPC span attributes as defined in the OpenTelemetry spec.
```

Reply:
Can we add a 

```
/// - SeeAlso: [OpenTelemetry: Semantic conventions for RPC spans](https://github.com/open-telemetry/opentelemetry-specification/blob/b70565d5a8a13d26c91fb692879dc874d22c3ac8/specification/trace/semantic_conventions/rpc.md) (as of August 2020)
```

Reply:
Yeah, that's a good addition. I'll add it to the other ones as well 👍